frontend/www/js/omegaup/components/badge/Listv2.test.ts

```diff
@@ -0,0 +1,19 @@
+import { shallowMount, mount } from '@vue/test-utils';
+import expect from 'expect';
+
+import T from '../../lang';
+import badge_list from './Listv2.vue';
```

Comment:
```suggestion
import badge_List from './Listv2.vue';
```

por consistencia con cómo se importa en otros lugares.